CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 * Allow specifying check-interval higher than 0 together with no limit configuration
 * Set default check-interval to 60s
+
+## 0.3.1
+
 * Change excluded principal definition to be semicolon-separated list
 
 ### Changes, deprecations and removals
 
-* **Important:** From 0.4.0 the excluded principals are configured as semicolon-separated list. In previous versions, the principals were
-configured using comma-separated list.
\ No newline at end of file
+* **Important:** From 0.3.1 the excluded principals are configured as semicolon-separated list. In previous versions, the principals were
+configured using comma-separated list.
+This fixes the issue with Distinguish Names, which were wrongly separated by the comma.
```

Comment:
```suggestion
* **Important:** 0.3.1 fixes a bug from 0.3.0 release where the excluded principals were separated by a comma instead of a semi-colon.
  If you use the `<xxx>` option, please update your configuration and use `;` as separators.
```

Reply:
0.2.0 does not seem to list any excluded principals option. I think that is new in 0.3.0, or?

Reply:
Yes, this is new in 0.3.0. 

We had this configuration option in a downstream fork (based on 0.2.0) for a long time but it didn't make it back upstream until now.

Reply:
ah. I see.